lib/broadside/cli/gli_commands.rb

```diff
@@ -0,0 +1,142 @@
+def add_tag_flag(cmd)
```

Comment:
the metaprogramming was nice before because we could public_send to a method in `Deploy`, but now it's different. `Command.bootstrap`, where `bootstrap` is the command would work, but  `Command.deploy_short` doesn't because the command name is `short`. In the end we don't have a strong need to use `public_send` so I think it's better just to remove it.